tests/test_pytorch_sparse_data_api.py

```diff
@@ -0,0 +1,331 @@
+"""Tests for TileDB integration with Pytorch sparse Data API."""
```

Comment:
Why no just rename it?

Reply:
Answering to the general and for this one here. I tried to match tests that we have in TF with the ones of Pytorch. The naming of the files are because again the test files for TF had a different naming than the ones of Pytorch. With this renaming we provide a cleaner mapping and file structure between two frameworks

Reply:
I see now I did rename it I don't know though why it passed like that in git. I followed the exact same way as the `tests/test_pytorch_dataloader_api.py -> tests/test_pytorch_data_api.py` for sparse files as well.

Reply:
1. Trying to 1-1 map tests between different frameworks might make sense in our case but generally doesn't. We could have completely different tests between them. It happens to have the same/similar tests because in both cases we work with generators.
2. Renaming to 'data_api' hides valuable first sight info. We used `Dataloader` naming convention in order to make clear that we work with iterables, i.e., PyTorch Dataloader by definition wraps an iterable around a Dataset. I would better try to find a better name for the Tensorflow-Keras case.
3. Deleting and re-writing the whole thing makes review difficult, i.e, I cannot just check the differences.